sdk/core/core-http/src/policies/bearerTokenAuthenticationPolicy.ts

```diff
@@ -46,6 +47,15 @@ export function bearerTokenAuthenticationPolicy(
  *
  */
 export class BearerTokenAuthenticationPolicy extends BaseRequestPolicy {
+  private tokenRefresher: AccessTokenRefresher;
+
+  /**
+   * The automated token refresh will only start to happen at the
+   * expiration date minus the value of timeBetweenRefreshAttemptsInMs,
+   * which is by default 30 seconds.
+   */
+  private timeBetweenRefreshAttemptsInMs: number = 30000;
```

Comment:
nit: no need to annotate the type as number

Reply:
also why make this a class member? It seems to be constant, in which case it could be pulled out into a module const

Reply:
I moved it to a module const here: https://github.com/Azure/azure-sdk-for-js/pull/10085/commits/16c88f5e2431c0f1a64ec1b44656d51ec1cbb709